libraries/GCS_MAVLink/GCS.h

```diff
@@ -1005,7 +1005,7 @@ class GCS_MAVLINK
     virtual void handle_manual_control_axes(const mavlink_manual_control_t &packet, const uint32_t tnow) {};
 
     void handle_vicon_position_estimate(const mavlink_message_t &msg);
-    void handle_vision_position_estimate(const mavlink_message_t &msg);
+    void handle_vision_pose_estimate(const mavlink_message_t &msg);
```

Comment:
Let's leave this one as-is because it matches the mavlink message being consumed.

Reply:
Done